core/templates/dev/head/services/ExplorationHtmlFormatterService.js

```diff
@@ -33,6 +33,7 @@ oppia.factory('ExplorationHtmlFormatterService', [
           extensionTagAssemblerService.formatCustomizationArgAttrs(
             element, interactionCustomizationArgSpecs));
         element.attr('on-submit', 'submitAnswer(answer, rulesService);');
+        element.attr('current-card', 'activeCard');
```

Comment:
I think this is too much info to pass around? It seems to me like just passing in correctAnswer would be enough (and only doing so when the answer is actually correct, in terms of the user moving on to the next card).

Reply:
How about I just pass the last input he gave the image before new card was available (presumably the correct one) ?

Reply:
That's fine (and I think that's what I meant).

Reply:
Oh, Ok.

Reply:
Done